cmd/livepeer_cli/wizard_transcoder.go

```diff
@@ -153,8 +167,11 @@ func (w *wizard) setOrchestratorConfig() {
 
 	val := w.getOrchestratorConfigFormValues()
 
-	httpPostWithParams(fmt.Sprintf("http://%v:%v/setOrchestratorConfig", w.host, w.httpPort), val)
-	// TODO we should confirm if the transaction was actually sent
+	result := httpPostWithParams(fmt.Sprintf("http://%v:%v/setOrchestratorConfig", w.host, w.httpPort), val)
+
+	if result == "" {
+		return
```

Comment:
```suggestion
	if result != "" {
	fmt.Printf("Error applying configuration: %s\n", result)
		return
```

Reply:
In this particular case `result` will be empty in case of success and will contain error string in case of error.

Reply:
fixed in cd54636